persistent_binderhub/files/jupyterhub/persistent_bhub_config.py

```diff
@@ -13,6 +13,15 @@
 from jupyterhub.apihandlers.users import admin_or_self
 from jupyterhub.apihandlers.base import APIHandler
 from kubespawner import KubeSpawner
+from traitlets import List
+
+DEFAULT_PROVIDERS = (
+    {"prefix": "gh", "name": "GitHub", "hostname": "github.com"},
+    {"prefix": "git", "name": "Git", "hostname": None},
+    {"prefix": "gl", "name": "GitLab", "hostname": "gitlab.com"},
+)
```

Comment:
Could you also add `Gist` here?
```
{"prefix": "gist", "name": "Gist", "hostname": "gist.github.com"},
```

Reply:
done. force pushed into the previous commit